randovania/layout/base/base_configuration.py

```diff
@@ -62,6 +63,7 @@ class BaseConfiguration(BitPackDataclass, JsonDataclass, DataclassPostInitTypeCh
     single_set_for_pickups_that_solve: bool
     staggered_multi_pickup_placement: bool
     check_if_beatable_after_base_patches: bool
+    all_obtainable: AllObtainableOption
```

Comment:
please add an entry in `dangerous_settings()` further down this file for the all pickups mode